ui/StatusQ/src/StatusQ/Controls/StatusTagItem.qml

```diff
@@ -68,49 +68,52 @@ Rectangle {
 
     width: tagRow.implicitWidth + 2 * d.tagMargins
     height: 30
-    color: d.getTagColor(root.isReadonly)
-    radius: 8
-    Row {
-        id: tagRow
-        height: parent.height
-        anchors.left: parent.left
-        anchors.leftMargin: d.tagMargins
-        anchors.rightMargin: d.tagMargins
-        spacing: 2
-
-        StatusIcon {
-            visible: root.icon
-            anchors.verticalCenter: parent.verticalCenter
-            color: Theme.palette.indirectColor1
-            width: root.icon ? d.tagIconsSize : 0
-            height: d.tagIconsSize
-            icon: root.icon
-        }
-        StatusBaseText {
-            id: nameText
-            anchors.verticalCenter: parent.verticalCenter
-            color: Theme.palette.indirectColor1
-            font.pixelSize: 15
-            text: root.text
-        }
-        StatusIcon {
-            id: closeIcon
-            visible: !root.isReadonly
-            anchors.leftMargin: d.tagMargins
-            anchors.verticalCenter: parent.verticalCenter
-            color: Theme.palette.indirectColor1
-            width: d.tagIconsSize
-            height: d.tagIconsSize
-            icon: "close"
-        }
+    background: Rectangle {
+        color: d.getTagColor(root.isReadonly)
+        radius: 8
     }
+    contentItem: Item {
+        Row {
+            id: tagRow
+            height: parent.height
+            anchors.left: parent.left
+            anchors.leftMargin: d.tagMargins
+            anchors.rightMargin: d.tagMargins
```

Comment:
How does the rightMargin work, there's no right anchor

Reply:
updated